### DISCUSSION TITLE
Received ”Your free GitHub Copilot preview is ending soon“, even though I'm eligible to use it free

### DISCUSSION THREAD

**Discussion by pan93412** [community/community#30116] [Copilot Conversations]
<img width="825" alt="截圖 2022-08-20 上午12 38 54" src="https://user-images.githubusercontent.com/28441561/185666473-82996d14-c665-4650-aa4d-7eee66db2c00.png">

<img width="834" alt="image" src="https://user-images.githubusercontent.com/28441561/185666530-fe132594-dfbe-452d-9d4a-7b63312111a9.png">

<img width="853" alt="image" src="https://user-images.githubusercontent.com/28441561/185666923-385b4b47-8ee3-418f-995f-b50adeeb7761.png">

The warning bar and the mail are really ambiguous – I don't know what further actions I can do for this warning, as I can't even find somewhere to subscribe my Copliot 🌚

**Comment by pan93412** [comment]
I found that it is duplicate of #30048, and it was confirmed as a bug.

**Comment by veverkap** [comment]
@pan93412 can you please check your settings again? We just pushed out a fix. 🤞

**Reply by pan93412** [reply]
It is fixed, thanks!
